common/scala/src/main/resources/application.conf

```diff
@@ -162,5 +162,9 @@ whisk {
         role = "*" //see http://mesos.apache.org/documentation/latest/roles/#associating-frameworks-with-roles
         failover-timeout = 0 seconds  //Timeout allowed for framework to reconnect after disconnection.
         mesos-link-log-message = true //If true, display a link to mesos in the static log message, otherwise do not include a link to mesos.
+        constraints = [] //placement constraint strings to use for managed containers e.g. ["att1 LIKE v1", "att2 UNLIKE v2"]
+        blackbox-constraints = [] //placement constraints to use for blackbox containers
```

Comment:
I'm wondering whether the difference is in `managed containers` vs `blackbox containers` to decide where to spin up new action containers, or if it should be in some other place. I.e if I want to run containers on `gpu` VMs vs regular VMs, regardless of them being `blackbox` or not, where can we define this ? The reason I'm asking is b/c if the decision point should be influenced by other factors, maybe we don't _have_ to differentiate here between `blackbox` vs `non-blackbox` containers ?